runtime/js/40_testing.js

```diff
@@ -186,6 +186,14 @@ finishing test case.`;
     ArrayPrototypePush(tests, testDef);
   }
 
+  function formatFailure(error) {
+    if (error.errors) {
+      return { failed: error.errors.map(error => inspectArgs([error])).join("\n") };
```

Comment:
Blank line in-between?

```suggestion
      return { failed: error.errors.map(error => inspectArgs([error])).join("\n\n") };
```